playwright-tests/e2e/utils/hyphenize.ts

```diff
@@ -0,0 +1,3 @@
+export const hyphenize = (str: string) => {
+    return str.split(" ").join("-");
+};
```

Comment:
An arrow function returns by default if not wrapped in a block.
```suggestion
export const hyphenize = (str: string) => str.split(" ").join("-");
```